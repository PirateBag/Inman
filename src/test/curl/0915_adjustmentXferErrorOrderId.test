Try out some errors.
=== JSON ===
{ "rows" : [
             { "amount" : "3.00", "itemId" : "17", "orderId" : "9999", "orderType" : "PO", "effectiveDate" : "2025-0728", "adjustmentType" : "XFER", "crudAction" : "INSERT" }

 ] }