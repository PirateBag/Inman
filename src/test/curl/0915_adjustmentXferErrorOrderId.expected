{
  "responseType" : null,
  "errors" : [ {
    "key" : 0,
    "message" : "Adustment item ref (17) not found"
  } ],
  "data" : [ ]
}